apps/api/resources/api/ViewCollection.ttl

```diff
@@ -36,4 +40,12 @@ PREFIX hydra: <http://www.w3.org/ns/hydra/core#>
           hydra:required true ;
         ] ;
     ] ;
+  knossos:preprocessPayload
+    [
+      code:implementedBy
+        [
+          a code:EcmaScriptModule ;
+          code:link <file:apps/api/lib/view/create.js#importMetadata>
+        ] ;
+    ] ;
```

Comment:
This is [creta resource hook](https://creta.hypermedia.app/#/advanced/hooks?id=preprocess-hooks) which will be called when sending a `POST` request to create new view.